### PR TITLE
feat(refactor): item repository extends mutable repository

### PR DESCRIPTION
**Pull request by ReidyT**

### CODE REVIEW

**Review by Defferrard — APPROVED** [review]
LGTM ! Goob job ! I've just an interrogation !

File: src/services/auth/plugins/passport/strategies/jwtApps.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default (
           });
         } catch (err) {
           // Exception occurred while fetching item
-          // itemRepository.get() can fail for many reasons like the item was not found, database error, etc.
+          // itemRepository.getOne() can fail for many reasons like the item was not found, database error, etc.
```
[the remark this answers is not in the document]
Comment (ReidyT) [reply src/services/auth/plugins/passport/strategies/jwtApps.ts:61]: In fact, I haven't made this change now for the Item, but I'll do it right now!

File: src/repositories/errors.ts [hunk 2]
```diff
@@ -64,3 +64,13 @@ export class DeleteException extends RepositoryException {
     super(message, 'DeleteException');
   }
 }
+
+// TODO: use entity class when rebase on main!
```
Comment (ReidyT) [inline src/repositories/errors.ts:68]: Update before merge!

File: src/services/auth/plugins/passport/strategies/jwtApps.ts [hunk 3]
```diff
@@ -58,7 +58,7 @@ export default (
           });
         } catch (err) {
           // Exception occurred while fetching item
-          // itemRepository.get() can fail for many reasons like the item was not found, database error, etc.
+          // itemRepository.getOne() can fail for many reasons like database error, etc.
```
Comment (ReidyT) [inline src/services/auth/plugins/passport/strategies/jwtApps.ts:61]:
```suggestion
          // itemRepository.getOneOrThrow() can fail for many reasons like the item was not found, database error, etc.
```

**Pull request merged**